tests/snappi/pfc/files/helper.py

```diff
@@ -71,8 +78,17 @@ def run_pfc_test(api,
                   'Fail to get ID for port {}'.format(dut_port))
 
     """ Rate percent must be an integer """
-    test_flow_rate_percent = int(TEST_FLOW_AGGR_RATE_PERCENT / len(test_prio_list))
     bg_flow_rate_percent = int(BG_FLOW_AGGR_RATE_PERCENT / len(bg_prio_list))
+    test_flow_rate_percent = int(TEST_FLOW_AGGR_RATE_PERCENT / len(test_prio_list))
+
+    if headroom_test_params is None:
+        unique_name_required = False
```

Comment:
Why we need unique names?

Reply:
the headroom test is run in a for loop, and each time the snappi config is setup we create flow names, which is not discarded when one run of the for loop finishes. The test will complain that there is a duplicate name for a flow on the second loop instance and so forth, so we need unique names for the flows since the config is not discarded after each run; only once the whole test finishes.